lib/facebook.js

```diff
@@ -181,18 +181,13 @@ export class Chat {
                 attachment: {
                     type: 'template',
                     payload: {
-                        'template_type': 'list',
-                        'top_element_style': topElementStyle,
+                        'template_type': 'generic',
```

Comment:
this is not acceptable. we need a new function and all uses need to be switched to the new function. the original `sendList` needs to stay or be deleted if it's not used anymore.

Reply:
We now have a function `sendGenericList` used in all list cases using `listElement` as its elements.